platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -188,6 +189,42 @@ public double getScreenRefreshRate(double fallback) {
 		return fallback;
 	}
 
+	public boolean isHdrSupported() {
+		Configuration configuration = activity.getResources().getConfiguration();
+		if (configuration != null) {
+			return configuration.isScreenHdr();
```

Comment:
The `isScreenHdr()` function is only available in Android SDK 26+, but Godot supports 24+ by default (as of 4.4). Calling this function will crash on 24 and 25 (Android 7.0 and 7.1 respectively).